gson/src/main/java/com/google/gson/stream/JsonReader.java

```diff
@@ -304,8 +304,6 @@ public JsonReader(Reader in) {
    *       prefix</a>, <code>")]}'\n"</code>.
    *   <li>Streams that include multiple top-level values. With strict parsing,
    *       each stream must contain exactly one top-level value.
-   *   <li>Top-level values of any type. With strict parsing, the top-level
```

Comment:
This and the changes below are from #1609 and the comments there.

This specific sentence was outdated because Gson already supports top-level values of other types, even in strict mode.